### PR TITLE
[RPA-334] Processes view rererendering 

### PR DESCRIPTION
**Pull request by kzuraniewski**

### CODE REVIEW

File: runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBrowseView/ProcessList/ProcessList.tsx [hunk 1]
```diff
@@ -49,19 +49,30 @@ const ProcessList: VFC = () => {
...
             />
             <ProcessPageProvider {...{
                 pageSize, setPageSize, search, searchField, page, setPage
             }}
             >
-                {displayMode === ProcessListDisplayMode.GRID
-                    ? <GridView />
-                    : <ProcessTable onAdvancedSearchChange={handleAdvancedSearch} />}
+                <If condition={displayMode === ProcessListDisplayMode.GRID}>
+                    <If condition={loading}>
```
Comment (preludium) [inline runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBrowseView/ProcessList/ProcessList.tsx]: you can merge this 2 If components, just use else prop

**Review by preludium — APPROVED** [review]

File: runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBrowseView/ProcessList/ProcessList.tsx (hunk 1 above)
Comment (kzuraniewski) [reply runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBrowseView/ProcessList/ProcessList.tsx]: done

File: runbotics/runbotics-orchestrator-ui/src/hooks/useLoading.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        setShowLoading(true);
+
+        const id = setTimeout(() => {
+            setIsDelayPassed(true);
+            clearTimeout(id);
+            console.log('timeout');
+            
+        }, delay);
+
+        console.log('1', isLoading, delay);
```
Comment (FranekMarciniak) [inline runbotics/runbotics-orchestrator-ui/src/hooks/useLoading.ts]: Remove console log

File: runbotics/runbotics-orchestrator-ui/src/hooks/useLoading.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        
+    }, [isLoading, delay]);
+
+    useEffect(() => {
+        setShowLoading(isLoading);
+
+        console.log('2', isDelayPassed);
```
Comment (FranekMarciniak) [inline runbotics/runbotics-orchestrator-ui/src/hooks/useLoading.ts]: same here

File: runbotics/runbotics-orchestrator-ui/src/hooks/useLoading.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            return;
+        }
+
+        setIsDelayPassed(false);
+        setShowLoading(true);
+
+        const id = setTimeout(() => {
+            setIsDelayPassed(true);
+            clearTimeout(id);
+            console.log('timeout');
```
Comment (FranekMarciniak) [inline runbotics/runbotics-orchestrator-ui/src/hooks/useLoading.ts]: same here

**Review by FranekMarciniak — APPROVED** [review]

**Pull request merged**